README.md

```diff
@@ -368,7 +368,7 @@ Parameters of the driver script, `abyss-pe`
  * `q`: minimum base quality [`3`]
  * `s`: minimum unitig size required for building contigs (bp) [`200`]
  * `S`: minimum contig size required for building scaffolds (bp) [`s`]
- * `t`: minimum tip size (bp) [`2k`]
+ * `t`: maximum length of dangling edges to trim [`k`]
```

Comment:
The question of `2*k` vs `k` is a matter of units. `k` *k*-mers are pruned, which is `2*k-1` bp. I'm not sure which unit is used by ABySS.